### ISSUE TITLE
Re-Running activity_classifier.create on the same exact training dataset produces models with different accuracies.

### ISSUE THREAD

**Issue by velociraptor111** [apple/turicreate#2081]
I am splitting my dataset using a seed value, so I know that whenever I run the statement it will create the same training and test set 
`train, test = tc.activity_classifier.util.random_split_by_session(data, session_id='session_id', fraction=0.95,seed=4)` 

however, whenever I recreate the model like so
`model = tc.activity_classifier.create(train, session_id='session_id', target='label', prediction_window=10,max_iterations=200)` it will produce different accuracies when evaluated `metrics = model.evaluate(test)
print(metrics['accuracy'])`

I'm not sure how turicreate activity classifier works in the background, but can this be due to the random initialization of weights when training the neural network? If so, how can we seed this random weight initialization value as well to give better consistency when training the model?

**Comment by velociraptor111** [comment]
After doing further investigation, we can see that , even for the same exact training data, the loss values are different

Experiment 1:
![Screen Shot 2019-06-24 at 9 22 24 AM](https://user-images.githubusercontent.com/19272937/60022834-960e6d00-9662-11e9-869a-091acf302eb1.png)

Experiment 2:
![Screen Shot 2019-06-24 at 9 29 55 AM](https://user-images.githubusercontent.com/19272937/60022869-a888a680-9662-11e9-9263-3222c22b4d5c.png)

**Comment by fareeha** [comment]
It may help to fix the mxnet seed, for example:

import mxnet as mx
mx.random.seed(0)

**Comment by TobyRoseman** [comment]
@velociraptor111 - Are you still seeing this behavoir with TuriCreate 6.1?

**Comment by velociraptor111** [comment]
Hi sorry , I decided to use a different technology stack instead. Closing, this issue now.
